src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -117,6 +123,7 @@ const convertInclude = async (includeBlock, ...restParams) => {
     type,
     model: {
       href,
+      indexOfInclude,
```

Comment:
Do we want to still add these indexes irrespective of what response the include fetch call returns or if it's a supported type or not ?

Reply:
This would result in potentially missing indexes if include 0 was supported, include 1 wasn't and include 2 was. I don't see this as an issue personally, the important thing is the process is determinitistic, so we couldn't use UUIDs for example.